src/inspector_io.cc

```diff
@@ -27,20 +27,36 @@ using v8_inspector::StringView;
 template<typename Transport>
 using TransportAndIo = std::pair<Transport*, InspectorIo*>;
 
-std::string GetProcessTitle() {
-  // uv_get_process_title will trim the title if it is too long.
+static std::string GetProcessTitle() {
```

Comment:
Thanks, I didn't notice that, I'll review all the statics for this (so you don't have to comment on each one).